infra/docker/server.Dockerfile

```
@@ -1,48 +1,56 @@
 # Build stage
 FROM node:20-alpine AS builder
+# Install openssl for Prisma
 RUN apk add --no-cache openssl && \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)